app/views/recipes/_template_field.html.erb

```diff
@@ -0,0 +1,21 @@
+<div data-controller="dynamic-form-field visibility"
+     data-dynamic-form-field-index-value="<%= form.index %>"
+     class="grid grid-cols-3 gap-6">
+  <div class="col-span-3">
+    <div class="flex rounded-md shadow-sm relative">
+      <%= form.text_field :name,
+                          class: "block w-full flex-1 rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm",
+                          placeholder: "Eggs",
+                          data: {
+                            dynamic_form_field_target: "field"
+                          } %>
+      <button data-action="visibility#remove"
```

Comment:
Because we know for a fact that a template can never be a persisted object, the logic here is unnecessary. The only possible action to delete this is to remove it from the dom so it just uses the stimulus visibility controller